### PR TITLE
Robga/exemption

### PR DESCRIPTION
**Pull request by robga**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

Add policy exemption preview. 

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by robga** [comment]
The latest error seems caused by the exemption file is referencing the latest azure-rest-api-specs/specification/common-types/resource-management/v2/types.json
The rest of the policy artifacts in the same package are referencing the older version azure-rest-api-specs/specification/common-types/resource-management/v1/types.json, which contains a bug. 
I updated the rest of policy artifacts in another PR: https://github.com/Azure/azure-rest-api-specs/pull/10617
I will make sure to check in that one first and merge the exemption to the newer package in readme.md.

**Comment by raych1** [comment]
@robga , please fix the lintDiff check failures?

**Comment by robga** [comment]
> @robga , please fix the lintDiff check failures?

@raych1, I will need to merge another PR: #10617 first. Then I will move this file to the newer package in readme.md as I explained in the earlier comment. 

> The latest error seems caused by the exemption file is referencing the latest azure-rest-api-specs/specification/common-types/resource-management/v2/types.json
> The rest of the policy artifacts in the same package are referencing the older version azure-rest-api-specs/specification/common-types/resource-management/v1/types.json, which contains a bug.
> I updated the rest of policy artifacts in another PR: #10617
> I will make sure to check in that one first and merge the exemption to the newer package in readme.md.

File: specification/resources/resource-manager/Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+        "metadata": {
+          "type": "object",
+          "description": "The policy exemption metadata. Metadata is an open ended object and is typically a collection of key value pairs."
+        }
+      },
+      "description": "The policy exemption properties."
+    },
+    "PolicyExemption": {
+      "properties": {
+        "properties": {
```
Comment (pilor) [inline specification/resources/resource-manager/Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json:444]: make "properties" required on the PolicyExemption definition

**Review by pilor — APPROVED** [review]
1 small comment

**Comment by robga** [comment]
@raych1 , I have fixed the error on the PR.

**Review by raych1 — APPROVED** [review]

**Pull request merged**